win32/src/PySECURITY_ATTRIBUTES.cpp

```diff
@@ -106,9 +104,9 @@ PySECURITY_ATTRIBUTES::PySECURITY_ATTRIBUTES(void)
     m_sa.nLength = sizeof(SECURITY_ATTRIBUTES);
     m_obSD = new PySECURITY_DESCRIPTOR(SECURITY_DESCRIPTOR_MIN_LENGTH);
     m_sa.lpSecurityDescriptor = ((PySECURITY_DESCRIPTOR *)m_obSD)->GetSD();
-    // On win95/98/me (or any platform that doesn't have NT security) the
-    // initialization of the SECURITY_DESCRIPTOR should fail, leaving the
-    // sd NULL.
+    // On platforms that don't have NT security,
+    // the initialization of the SECURITY_DESCRIPTOR should fail,
+    // leaving the sd NULL.
```

Comment:
I wonder if I should just essentially revert https://github.com/mhammond/pywin32/commit/b9e3c3f095f7d295996624c77ff86a51b233db0d

Or if there's valid use-cases for all the checks added there (CC @trentm @rupole )